docs/EN/Configuration/Accu-Chek-Combo-Pump.md

```diff
@@ -86,13 +86,12 @@ Please be aware that this is not complete list and reflects personal user experi
 
 ![Screenshot of insulin cartridge settings](../images/combo/combo-insulin-settings.png)
 
-- Install AndroidAPS as described in the [AndroidAPS wiki](http://wiki.AndroidAPS.org) and use the `combo` branch.
+- Install AndroidAPS as described in the [AndroidAPS wiki](https://androidaps.readthedocs.io/)
 - Make sure to read the wiki to understand how to setup AndroidAPS.
 - Select the MDI plugin in AndroidAPS, not the Combo plugin at this point to avoid the Combo
   plugin from interfering with ruffy during the pairing process.
-- Follow the link http://ruffy.AndroidAPS.org and clone ruffy via git. Use the same branch as you use for
-  AndroidAPS, right now that's the `combo` branch, later on there will be the regular `master` and `dev` branches.
-- Install ruffy and use it to pair the pump. If it doesn't work after multiple attempts, switch to the `pairing` branch, pair the pump and then switch back the original branch.
+- Clone ruffy via git from https://github.com/MilosKozak/ruffy. At the moment, the primary branch is the `combo` branch, in case of problems you might also try the 'pairing' branch (see below).
```

Comment:
due to punctuation this link does not work, so to avoid this you can use a markdown link like `... from [MilosKozak/ruffy](https://github.com/MilosKozak/ruffy). At ...`

Reply:
Thanks for your feedback, I have updated some hypertext links and commited the file again.